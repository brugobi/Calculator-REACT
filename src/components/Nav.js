import React from 'react';

function Nav() {
  return (
    <nav className="navbar is-transparent" role="navigation" aria-label="main navigation">

      <div className="navbar-brand">
        <div className="navbar-item">
          <div className="brand">Math Magicians</div>
        </div>
      </div>

      <div className="navbar-end">
        <div className="navbar-item">
          <a href="/">Home</a>
        </div>
        <div className="navbar-item">
          <a href="/">Calculator</a>
        </div>
        <div className="navbar-item">
          <a href="/">Quote</a>
        </div>
      </div>
    </nav>
  );
}

export default Nav;
