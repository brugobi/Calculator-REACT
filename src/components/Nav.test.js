import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Nav from './Nav';

describe('Nav component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<BrowserRouter><Nav /></BrowserRouter>);
    expect(tree).toMatchSnapshot();
  });
});
