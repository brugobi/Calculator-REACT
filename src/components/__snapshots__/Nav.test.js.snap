// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches the snapshot 1`] = `
<nav
  aria-label="main navigation"
  className="navbar"
  role="navigation"
>
  <div
    className="navbar-brand"
  >
    <div
      className="navbar-item"
    >
      <div
        className="brand font-orange font20px ZenDotsFont"
      >
        Math Magicians
      </div>
    </div>
  </div>
  <div
    className="navbar-end font-orange ZenDotsFont"
  >
    <div
      className="navbar-item"
    >
      <a
        className="nav-link font-orange"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </div>
    <div
      className="navbar-item"
    >
      <a
        className="nav-link font-orange"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </div>
    <div
      className="navbar-item"
    >
      <a
        className="nav-link font-orange"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </div>
</nav>
`;
